force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1361,6 +1361,38 @@
         <shortDescription>EPAddDependentTask</shortDescription>
         <value>Add Dependent Task</value>
     </labels>
+    <labels>
+        <fullName>donorsGivingSummary</fullName>
+        <categories>Donors</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Component title</shortDescription>
+        <value>Giving Summary</value>
+    </labels>
+    <labels>
+        <fullName>donorsLifetime</fullName>
```

Comment:
@lparrott What do you think about having all of the custom labels for this project prefixed with `donorGS` or `donorGivingSummary` rather than just `donors`?

Reply:
`donorGS` is good with me, but I'm not picky 😅